src/views/admin-users-listing.phtml

```diff
@@ -62,6 +62,33 @@ require_once('partials/head.phtml');
                 </tbody>
             </table>
         </div>
+
```

Comment:
From what I see in the Controller there is no value provided for 'usersNumber'. Only user, page and pages.